cloud-ide.md

```diff
@@ -1 +1,267 @@
+
+<img width="600" alt="codenvy file pane" src="images/codenvy/ui-breakdown/the_file.png" />
+
+You can import existing projects, create your own from scratch, or use a template. If you have no files, you can create them manually or use a template. 
+
+- To use a template, click **Create Project...**, select the **web-rails-simple** example, name your project, and click **Create**:
+
+  <img width="600" alt="codenvy example creation" src="images/codenvy/ui-breakdown/create-example.png" />
+
+The folder icon indicates andy type of folder, while extension-specific icons represent individual files.
+
+- Double-click on a folder:
+
+  <img width="500" alt="codenvy folder click" src="images/codenvy/ui-breakdown/the_folder.png" />
+
+- The contents of the file will populate the pane to the right:
+
+  <img width="500" alt="codenvy file pane" src="images/codenvy/ui-breakdown/the_click.png" />
+
+### Edit File Pane
+
+The Edit File pane shows the contents of a file and allows you to edit it. Changes to files are saved automatically. See the description of the coding environment below.
+
+  <img width="600" alt="codenvy file pane" src="images/codenvy/ui-breakdown/the_edit.png" />
+
+By clicking in the Edit File pane, you can adjust the contents of a file. To save the changes, you can hold **command+S** (**CTRL+S** for Windows), or go to the top menu and click **File > Save**.
+
+We'll be using this coding environment extensively. In the next lesson, we'll cover how to interact with the coding environment as well as some of the most important commands within the coding environment. 
+
+## Setting Up Projects
+
+In this section, we'll walk through setting up the first example application we'll be building - _splurty_. 
+
+> Note: These concepts apply to all projects we will be creating (and almost all of this you would have to do as part of the project setup even if you _weren't_ using a cloud IDE).
+
+Follow these steps to set up the project, using [the terminal](#the-terminal) and the [the edit file pane](#edit-file-pane) provided in the browser:
+
+### Step 1: Creating the App
+
+We can use `rails new` to create the new app, though we'll need to make sure we create the app in the right folder.
+
+1. For us to be able to see the code in the editor, we'll need to be in projects folder any time we create a project. In the terminal, type the following: 
+    ```sh
+    cd projects
+    ```
+
+2. We can create a new rails app, install dependencies (done as part of `rails new`), and move into our new project folder using the following commands:
+   ```sh
+   rails new test2
+   cd test2
+   ```
+ 
+### Step 2: Setting up the Database (Postgres)
+
+Next, we need to set up our application to use postgres:
+
+1. Ensure postgres is running:
+    ```sh
+    service postgresql start
+    ```
+2. Open up the `config/database.yml` file in your editor and change it so that it looks exactly like the below: 
```

Comment:
maybe we can make quick note that these files are senstitive to whitespace?  I common problem when editting the database.yml is to change whitespace.